### ISSUE TITLE
Add "Landlord Open to Customization" to seed data for custom fields

### ISSUE THREAD

**Issue by jliu1016** [CivicTechWR/accessible-housing-portal#408]
Discussed on Slack, Sean identified a need to indicate if a landlord is willing to alter a unit to help someone meet accessibility needs. This doesn't need to be filterable, and is 1 datapoint for the whole listing. Although this can be added via custom fields, we should just add it to the seeded custom fields since it seems evergreen.

Todo:

Write copy for the description
Add to seed data
